spec/models/miq_server/server_monitor_spec.rb

```diff
@@ -181,17 +181,6 @@
           expect(@miq_server.active_role_names.include?("database_operations")).to be_truthy
         end
 
-        it "should activate newly assigned limited region role" do
```

Comment:
This feels like it should stay but with a different candidate, unless it's already tested elsewhere.